Core/Src/spi_command_handler.c

```diff
@@ -11,6 +11,10 @@ extern uint8_t cam_to_nand_transfer_flag;
 uint32_t image_length; // Only here for testing purposes
 static uint32_t count = 0x0FFF0000;
 
+uint8_t sensor_mode = 0;
```

Comment:
this is hardcoded to take a vis image only?

Reply:
Sorry I think I have not rebase this branch properly to use the symbols from the iris_system.h